src/patternfly/components/ContextSelector/context-selector.scss

```diff
@@ -118,9 +130,35 @@
     }
   }
 
-  .pf-c-context-selector__toggle-icon {
-    margin-right: var(--pf-c-context-selector__toggle-icon--MarginRight);
-    margin-left: var(--pf-c-context-selector__toggle-icon--MarginLeft);
+  &.pf-m-plain {
+    --pf-c-context-selector__toggle--PaddingRight: var(--pf-c-context-selector__toggle--m-plain--m-text--PaddingRight);
```

Comment:
We typically try to have the vars match the CSS selectors, so we wouldn't want `--m-text` in the var name if it isn't in a `.pf-m-text` selector.

What you could do is move the `--pf-c-context-selector__toggle--m-plain--PaddingRight/Left` vars below up here, then write a nested `&.pf-m-text` selector and move these vars into that block.